### ISSUE TITLE
Whenever the party receives prosperity show an popup that notifies the party that they have receive prosperity

### ISSUE THREAD

**Issue by skahunter** [Lurkars/gloomhavensecretariat#410]
### Is your feature request related to a problem? Please describe.

You receive Porsperity from creating buildings and retiring heroes, but it's not shown on the screen so we always need to verify, did we get the prosperity or did we not.

### Describe the solution you'd like

I think showing a popup displaying: 

Rewarded
The party has been rewarded with 1 Prosperity.

This can be used if there are other moments where "hidden" unlocks happen. For example, stickers and new scenarios.

### Describe alternatives you've considered

An alternative could be that the prosperity is shown on the party sheet instead of the campaign manager.

**Comment by Lurkars** [comment]
Yes, we already spoke on Reddit. This is a good idea. 

Current state: Prosperity only gets added automatically from scenario rewards. 

In future: automatic prosperity on buildings and retirements and to not confuse from previous behavior, a pop up will show up with option to accept/decline and a setting to automatically accept/decline those dialogs.

**Comment by mentonin** [comment]
Even with auto accept, there could be a visual animation of "+1 prosperity" showing up and fading away, like video game damage or level up

**Comment by Lurkars** [comment]
I now changed a bit all this handling. In [v0.78.0](https://github.com/Lurkars/gloomhavensecretariat/releases/tag/v0.78.0) I added two new dialogs.
- when building / upgrading a building, the dialog for selecting resources now also list the bonuses like prosperity etc and opens the section when applied, the dialog stays anyways for now, but automatic adding those bonuses can be disabled by unchecking "Apply Building Bonuses" in settings
- on retirement, also just a new dialog opens up with applying the retirement bonuses, so very similar and can be disabled with "Apply Retirement Bonuses". The idea to make also this dialog is just with #366 in mind. If this will be developed, then the dialog will make much more sense.

So happy about feedback, I know that this won't give the complete automation as may wished, but then we can still keep this issue alive, otherwise we close, if solved with the newest changes.
